exercises/C-variables/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `greeting`
-
+let greeting="Hello World"
```

Comment:
Hi @Tony-devops ,
Generally, when you're writing a variable declaration like this, you need to add a `;` at the end. `;` is used to indicate the end of a statement in JS. Sometimes a `;` can be omitted but it's generally good practice to put a `;` at the end of statements.

Reply:
got it and same with sapcing. I need to shed the bad habits quickly. thanks